Makefile

```diff
@@ -28,10 +31,17 @@ build: doc dg
 	docker build --tag=${IMAGE_REPOSITORY} -f $(DOCKERFILE) .
 
 run: build
-	docker run -e MYHOSTNAME=localhost $(IMAGE_REPOSITORY)
+	docker run -e MYHOSTNAME=localhost -e PLAIN_AUTH $(IMAGE_REPOSITORY)
 
 test: build
-	cd tests; MODULE=docker URL="docker=${IMAGE_REPOSITORY}" DOCKERFILE="../$(DOCKERFILE)" VERSION=${VERSION} DISTRO=${DISTRO} mtf -l *.py
+	cd tests; MODULE=docker URL="docker=${IMAGE_REPOSITORY}" DOCKERFILE="../$(DOCKERFILE)" VERSION=${VERSION} DISTRO=${DISTRO} mtf-env-set
+	cd tests; MODULE=docker URL="docker=${IMAGE_REPOSITORY}" DOCKERFILE="../$(DOCKERFILE)" VERSION=${VERSION} DISTRO=${DISTRO} mtf --show-job-log *.py
+
+test-in-container: test-image
+	docker run --rm -ti -v /root/avocado:/root/avocado -v /run/docker.sock:/run/docker.sock:Z -v ${PWD}:/src ${TEST_IMAGE_NAME} "SELECTORS=${SELECTORS}"
```

Comment:
`-v /root/avocado:/root/avocado` this causes that you have to be __root__ to run this, but you can use also your current direcotru like `${PWD}/avocado:/root/avocado` or something similar